scripts/populate.js

```diff
@@ -3,7 +3,17 @@ var Chatty = (function(originalChatty){
 	var outputString = "";
 	var messagesArray = [];
 
-	
+	Chatty.getMessage = function () {
+		return messagesArray;
+	}
+
+	Chatty.newMessage = function () {
+		messagesArray.push(inputEl.value);
+		for (var i = 0; i < messagesArray.length; i++) {
+			outputString = messagesArray[i];
+		};
+			messageContainer.innerHTML += "<div id='message" + [i] + "' class='newMessages'>" + "User: " + outputString + "<button class='deleteButton'>Delete</button>" + "</div>";
```

Comment:
I think messageContainer is declared somewhere that I haven't looked yet, but I would declare (but not set the value of) a private variable in the iife called containerDiv or something similar. Since getMessage is going to receive an element ID everytime it's called I would have the function set the value of containerDiv to document.getElementById(elId) right before or after we push the string to the array.